testing/ef-tests/tests/tests.rs

```diff
@@ -1,8 +1,8 @@
-#![cfg(feature = "ef-tests")]
+// #![cfg(feature = "ef-tests")]
```

Comment:
```suggestion
#![cfg(feature = "ef-tests")]
```

you need to uncomment this